tests/test-st2bot-sigusr2.js

```diff
@@ -35,9 +36,35 @@ describe("SIGUSR2", function () {
   var debug_spy = sinon.spy(robot.logger, 'debug'),
     info_spy = sinon.spy(robot.logger, 'info');
 
+  beforeEach(function () {
+    // emulate ST2 API response
+    nock('http://localhost:9101')
```

Comment:
Seems the test worked before without Nock and emulating HTTP response.
What's the advantage of adding it now? Could be confusing a bit, giving assumption that specific HTTP response is required for `SIGUSR2` signal to work properly.

Reply:
It increases code coverage by taking paths in `loadCommands` that other tests do not.

I'll add a comment that it isn't required.